torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -3869,8 +3869,8 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::pointToPoint(
     return work;
```

Comment:
nit: shall we consolidate this `return work` with the one at the end? 
`collective()` function uses the follow style:
https://github.com/pytorch/pytorch/blob/edc3ca577e17286e5ad0a5e65ebaa377b0f82b72/torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp#L3408-L3416